EMS/core-bundle/src/Repository/FieldTypeRepository.php

```diff
@@ -18,6 +18,14 @@ public function __construct(Registry $registry)
         parent::__construct($registry, FieldType::class);
     }
 
+    /**
+     * @return FieldType[]
+     */
+    public function findAllNotDeleted(): array
+    {
+        return $this->findBy(['deleted' => false]);
+    }
+
```

Comment:
Useless in my head

Reply:
https://github.com/ems-project/elasticms/pull/788/commits/2db8fde453f09826dc8ed369f897d84c74cb479e